### DISCUSSION TITLE
Awesome App!

### DISCUSSION THREAD

**Discussion by lawnmower351** [Snaggly/KSW-ToolKit#18] [General]
This help really helped to add functionality to my Xtrons unit for my BMW M3.
I use the Options button to trigger my aftermarket front camera and the Nav button for zlink Android Auto.
The brightness feature works great.
Thanks,
Lou

**Comment by RSV4RR** [comment]
Hi guys I’m new to this my first android unit for Mercedes 
I’m trying to install this and I keep gettin an error 
App not installed package conflicts with existing 
Can someone please help me out on the install

TIA
